config/application.rb

```diff
@@ -36,6 +36,7 @@ class Application < Rails::Application
 
 Sentry.init do |config|
   config.breadcrumbs_logger = [:active_support_logger]
+  config.logger =  Logger.new(STDOUT)
```

Comment:
Should it be `Rails.logger`?

Reply:
at this point Rails.logger doesn't exist